src/views/Tag/Tags.vue

```diff
@@ -45,6 +39,18 @@
           <md-table-cell md-label="Color">
             <TagBoxChip :color="item.color" :text="item.color"></TagBoxChip>
           </md-table-cell>
+          <md-table-cell md-numeric>
+            <md-menu md-size="small" md-align-trigger>
+              <md-button md-menu-trigger class="md-icon-button">
+                <md-icon> more_horiz </md-icon>
+              </md-button>
+              <md-menu-content>
+                <md-menu-item @click="unsubscribe(item)">
+                  Unsubscribe
+                </md-menu-item>
+              </md-menu-content>
+            </md-menu>
+          </md-table-cell>
```

Comment:
The unsubscribe button shouldn't be displayed if the user is the author of the tag.